ergast_py/ergast.py

```diff
@@ -409,7 +409,7 @@ def get_constructor_standings(self) -> list[StandingsList]:
         Get a list of constructor standings from the current query
         """
         return self._get_items(self.requester.get_constructor_standings,
-                               self.type_constructor.construct_constructor_standings)
+                               self.type_constructor.construct_standings_lists)
```

Comment:
What about if we run
```python
mercedes = e.constructor('mercedes').season().get_constructor_standing()
print(mercedes)
```

Currently, we'll still print
```
ConstructorStanding(position=0, position_text=, points=0.0, wins=0, constructor=Constructor(constructor_id=, url=, name=, nationality=))
```
so we should probably update the singular versions of these functions to use `construct_standings_lists` too!